components/org.wso2.carbon.identity.api.server.secret.management/org.wso2.carbon.identity.api.server.secret.management.common/pom.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org).
```

Comment:
This should be changed (http://www.wso2.org) -> (http://www.wso2.com) as per the new license header. Check other places as well

Reply:
Refer mail "Re: License Header for WSO2 code"

Reply:
Addressed with commit [10818af](https://github.com/wso2/identity-api-server/pull/323/commits/10818afaea8628672ed76012026d308c49de0471)